ublox/src/ubx_packets/types.rs

```diff
@@ -17,6 +17,21 @@ pub struct Position {
     pub alt: f64,
 }
 
+/// Represents a world position in the ECEF coordinate system
+#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
+#[derive(Debug, Clone, Copy)]
+pub struct PositionECEF {
+    /// Logitude in degrees
+    pub x: f64,
```

Comment:
Hello,
the proposed documentation is wrong: these are ECEF coordinates not Longitude Latitude angles nor Altitude (as you correctly defined in `NavHpPosEcef`)